apis/cluster/composition.yaml

```diff
@@ -100,7 +129,7 @@ spec:
         kind: Usage
         spec:
           of:
-            apiVersion: aws.platform.upbound.io/v1alpha1
+            apiVersion: azure.platformref.upbound.io/v1alpha1
```

Comment:
khm, why/how did it work before with this bug? :D

Reply:
this usage is optional - so we didn't catch before

Reply:
yeah, it's still interesting that deletion ordering worked without it, apparently by accident :)